plugins/action/k8s_info.py

```diff
@@ -62,7 +62,14 @@ def run(self, tmp=None, task_vars=None):
 
         # find the file in the expected search path
         src = self._task.args.get('src', None)
+        remote_src = self._task.args.get('remote_src', False)
+
         if src:
+            if remote_src:
```

Comment:
Do we want to do something for `kubeconfig` as well? It's a bit awkward but that is another file that we look for locally but could be present remotely.